npdarrington/travel-tracker
# Nd bug post date

### What’s this PR do?  
- This PR fixes a bug when a user submits a new trip. The date that is submitted back to the API is one day less than the day they selected on the calendar.
 
### Where should the reviewer start?  
- Git pull this branch.
- Open index.js.
 
### How should this be manually tested?  
- Run `npm start` to start the server.
- Navigate to localhost:8080.
- Fill out the submission form and see if it is posting with the correct date.
 
### Any background context you want to provide?  
- N/A
 
### What are the relevant tickets?  
- Closes #74 
 
### Screenshots (if appropriate)
- N/A  
 
### Questions: 
- N/A
